journalbeat/reader/fields.go

```diff
@@ -19,69 +19,75 @@ package reader
 
 import "github.com/coreos/go-systemd/sdjournal"
 
+type fieldConversion struct {
+	name      string
+	isInteger bool
+	dropped   bool
+}
+
 var (
-	journaldEventFields = map[string]string{
+	journaldEventFields = map[string]fieldConversion{
```

Comment:
Hmm... I think I could use `schema` to transform the `Fields` of an `JournalEntry`, because it is a `map[string]string`. Thanks for the tip, I will check it out.